simulator/infra/types.h

```diff
@@ -73,7 +73,7 @@ using Addr = uint32;
 static const uint8  NO_VAL8  = 0xBA;
 static const uint16 NO_VAL16 = 0xf00d;
 static const uint32 NO_VAL32 = 0xdeadbeeful;
-static const uint64 NO_VAL64 = 0xfeedfacecafebeafull;
+static const uint64 NO_VAL64 = 0xdeadfacecafebeafull;
```

Comment:
The idea is to have no intersections between these values so one may understand where the value is originated from